bucket/filebot.json

```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://www.filebot.net/",
+    "description": "The ultimate TV and Movie Renamer",
```

Comment:
```suggestion
    "description": "Movies, TV Shows and Anime files organizer and renamer.",
```

Reply:
I'd keep the official branding, but if you prefer the one you write, then that's perfectly fine too.

Reply:
`The ultimate` is marketing stuff, which has anything to do with application description.

Reply:
I auto-generate manifests for brew / chocolatey / scoop / etc in my build:
```
package.synopsis: The ultimate TV and Movie Renamer
package.description: FileBot is the ultimate tool for organizing and renaming your Movies, TV Shows and Anime as well as fetching subtitles and artwork. It's smart and just works.
```
No worries. I'll just hard code description you like in my scoop template.

Reply:
You do not need to generate scoop manfiest in each build. All manifests are automatically updated on hour basis.  (https://github.com/ScoopInstaller/Excavator)

Reply:
As soon as PR is merged and new version is relased and identified, manifest will be updated in repository.

Reply:
Alright. That makes sense. Thanks!

Reply:
Your welcome.